ciao-image/datastore/posix.go

```
@@ -44,5 +44,15 @@ func (p *Posix) Write(ID string, body io.Reader) (int64, error) {
 func (p *Posix) Delete(ID string) error {
 	imageName := path.Join(p.MountPoint, ID)
 
-	return os.Remove(imageName)
+	_, err := os.Stat(imageName)
+	if err != nil {
+		if os.IsNotExist(err) {
+			err = nil
+		}
+		return err
+	}
+
+	os.Remove(imageName)
```

We're still ignoring the error from os.Remove